### DISCUSSION TITLE
how to call a function from another function in store

### DISCUSSION THREAD

**Discussion by SeongwoonHong** [pmndrs/zustand#857] [Q&A] [answered]
Hello!
I wonder how to do this
{
  AA: () => {
     set(....)
  },
  BB: () => {
      set(...)
      // call AA above
   },
   
}

**Comment by dai-shi** [comment, marked as answer]
See: https://github.com/pmndrs/zustand/discussions/854#discussioncomment-2302910

Now, I'm sure there's room for improvement in readme.md. 🤔
